{
  "Entries": [
    {
      "RequestUri": "https://api.loganalytics.io/v1/workspaces/40712a32-9002-4a78-a699-bd27fd863f86/query",
      "RequestMethod": "POST",
      "RequestHeaders": {
        "Accept": "application/json",
        "Authorization": "Sanitized",
        "Content-Length": "139",
        "Content-Type": "application/json",
        "traceparent": "00-7557c0b32ab5680f62aab388e17b64d7-eb984da336921eed-00",
        "User-Agent": "azsdk-net-Monitor.Query/1.3.0-alpha.20240205.1 (.NET 7.0.15; Microsoft Windows 10.0.22631)",
        "x-ms-client-request-id": "77eaefdc1aafcc3f8281988ee48a2916",
        "x-ms-return-client-request-id": "true"
      },
      "RequestBody": {
        "query": "set truncationmaxrecords=1; datatable (s: string) [\u0027a\u0027, \u0027b\u0027]",
        "timespan": "2024-01-22T00:00:00.0000000Z/P15D"
      },
      "StatusCode": 200,
      "ResponseHeaders": {
        "Access-Control-Allow-Origin": "*",
        "Access-Control-Expose-Headers": "Retry-After,Age,WWW-Authenticate,x-resource-identities,x-ms-status-location",
        "Age": "109",
        "Connection": "keep-alive",
        "Content-Length": "596",
        "Content-Type": "application/json; charset=utf-8",
        "Date": "Mon, 05 Feb 2024 23:14:51 GMT",
        "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
        "Vary": "Accept-Encoding",
        "Via": "1.1 draft-oms-7c485d9dfc-gf6fm",
        "X-Content-Type-Options": "nosniff"
      },
      "ResponseBody": {
        "tables": [
          {
            "name": "PrimaryResult",
            "columns": [
              {
                "name": "s",
                "type": "string"
              }
            ],
            "rows": [
              [
                "a"
              ]
            ]
          }
        ],
        "error": {
          "code": "PartialError",
          "message": "There were some errors when processing your query.",
          "details": [
            {
              "code": "EngineError",
              "message": "Something went wrong processing your query on the server.",
              "innererror": {
                "code": "-2133196797",
                "message": "The results of this query exceed the set limit of 1 records, so not all records were returned (E_QUERY_RESULT_SET_TOO_LARGE, 0x80DA0003). See https://aka.ms/kustoquerylimits for more information and possible solutions.",
                "severity": 2,
                "severityName": "Error"
              }
            }
          ]
        }
      }
    }
  ],
  "Variables": {
    "DateTimeOffsetNow": "2024-02-05T15:14:45.0377622-08:00",
    "LOGS_ENDPOINT": "https://api.loganalytics.io",
    "RandomSeed": "1530601106",
    "WORKSPACE_ID": "40712a32-9002-4a78-a699-bd27fd863f86"
  }
}
